### PR TITLE
update build tools to most recent ones

### PR DESCRIPTION
**Pull request by hannesa2**
It solves no issue, it's just to be up to date

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.1.0-beta1'
```
Comment (xiphirx) [inline build.gradle]: Lets not use a beta version when we dont need to.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Sat Jan 20 13:47:18 CET 2018
+#Wed Jan 31 15:25:02 CET 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (xiphirx) [inline gradle/wrapper/gradle-wrapper.properties]: 4.5 is the latest version.

Comment (hannesa2) [reply gradle/wrapper/gradle-wrapper.properties]: done

**Comment by xiphirx** [comment]
Thanks!

**Pull request merged**